### ISSUE TITLE
About TryMe

### ISSUE THREAD

**Issue by MasonKidder345** [Zrips/CMI#6037]
Just wanted to ask where could I suggest anything for TryMe because I can't find a repository for it

**Comment by mrfloris** [comment]
Yes, you can post your suggestion for tryme, if this is the wrong place ppl can be upset with me. Go ahead
